source/general/hdu-hierarchy.rst

```diff
@@ -0,0 +1,87 @@
+.. include:: ../references.txt
+
+.. _hdu-hierarchy:
+
+HDU hierarchy
+=============
+ 
+Following NASA's recommendation 
+(see `HFWG Recommendation R8 <http://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/general/ogip_94_006/ogip_94_006.html>`__), 
+a hierarchical classification is applied to each HDU within DL3 FITS files, 
+using the HDUCLASS and HDUCLASn keywords.
+
+Some useful links from other projects:
+
+* http://cxc.harvard.edu/contrib/arots/fits/content.txt
+* https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/ofwg_recomm/hduclas.html
+* http://www.starlink.rl.ac.uk/docs/sun167.htx/sun167se3.html
+* https://confluence.slac.stanford.edu/display/ST/LAT+Photons
+
+The current scheme used is the following:
+
+* HDUCLASS: General identifier of the data format. Recommended value: "OGIP"
+
+* HDUCLAS1: General type of HDU, currently: ``EVENTS``, ``GTI`` or ``RESPONSE``
+
+* HDUCLAS2: In case of ``RESPONSE`` type, refers to the IRF components stored within the HDU: ``EFF_AREA``, ``BKG``, ``EDISP`` or ``RPSF``
+
+* HDUCLAS3: In case of ``RESPONSE`` type, refers to the way the IRF component was produced (``POINT-LIKE`` or ``FULL-ENCLOSURE``)
+
+* HDUCLAS4: In case of ``RESPONSE`` type, refers to the name of the specific format
+
```

Comment:
The description of what to put for HDUDOC could be central, putting the Github URL.
For HDUVERS I would suggest to put "0.2" or "1.0" for now for all formats, and then on the general page describe that they can evolve differently, that it's not coupled to the global version from this spec.